package.json

```
@@ -10,7 +10,11 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "prettier-check": "prettier --check .",
-    "prettier-fix": "prettier --write ."
+    "prettier-fix": "prettier --write .",
+    "test": "vitest",
+    "test:watch": "vitest --watch",
+    "test:coverage": "vitest run --coverage",
+    "test:coverage:open": "vitest run --coverage && open coverage/index.html"
```

I don't think we should bloat scripts like this.
`vitest` already enters the watch mode in development environment and run mode in CI automatically, and `"test:coverage:open"` is a bit too much in my opinion.

```suggestion
    "test": "vitest",
    "test:coverage": "vitest run --coverage",
```

---

Of course~~ Let me adjust that.

---

Updated